### ISSUE TITLE
Upgrade from 3.2.1 to 4.0.0 causes FlywayException by using PostgreSQL

### ISSUE THREAD

**Issue by theexiile1305** [jobrunr/jobrunr#230]
### Discussed in https://github.com/jobrunr/jobrunr/discussions/224

<div type='discussions-op-text'>

<sup>Originally posted by **theexiile1305** September 27, 2021</sup>
Hello

First of all, thank you very much for this great project. 
After upgrading from `3.2.1` to `4.0.0` the following exception is thrown whenever I execute my integration tests in the CI-Pipeline. I am using Spring Boot, Flyway and a PostgreSQL within my web application.
```
java.lang.IllegalStateException: Failed to load ApplicationContext
            at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:132)
            at org.springframework.test.context.support.DefaultTestContext.getApplicationContext(DefaultTestContext.java:124)
            at org.springframework.test.context.junit.jupiter.SpringExtension.getApplicationContext(SpringExtension.java:283)
            at org.springframework.test.context.junit.jupiter.SpringExtension.resolveParameter(SpringExtension.java:269)
            at org.junit.jupiter.engine.execution.ExecutableInvoker.resolveParameter(ExecutableInvoker.java:216)
            ... 70 more
            Caused by:
            org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'flywayInitializer' defined in class path resource [org/springframework/boot/autoconfigure/flyway/FlywayAutoConfiguration$FlywayConfiguration.class]: Invocation of init method failed; nested exception is org.flywaydb.core.api.FlywayException: Found non-empty schema(s) "public" but no schema history table. Use baseline() or set baselineOnMigrate to true to initialize the schema history 
```

Do you have any idea why this error happens? 

Thank you very much in advance!
</div>

**Comment by rdehuyss** [comment]
This is unrelated to JobRunr - the stacktrace does not contain anything regarding JobRunr.

**Comment by BastiOfBerlin** [comment]
You should indicate that this fix only works with Spring Boot 2.5+, making jobrunr-spring-boot-starter basically dependent on Spring Boot 2.5+
